src/main/java/dan200/computercraft/core/lua/LuaJLuaMachine.java

```diff
@@ -412,6 +415,7 @@ public void execute()
                                             }
                                             catch( Throwable t )
                                             {
+                                                ComputerCraft.log.error( "Error running task", t);
```

Comment:
The issue here is *well behaved peripheral* - every so often people write buggy code which ends up throwing NPEs or other runtime exceptions. Whilst the exception is returned to the user, it doesn't include useful debugging information such as line numbers or stack traces - this PR is designed to make it easier for users/mod developers to track down and so solve issues in peripherals.